interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -216,6 +222,8 @@ void parseArgs(String[] args) throws Exception {
         soakResponseSize = Integer.parseInt(value);
       } else if ("additional_metadata".equals(key)) {
         additionalMetadata = value;
+      } else if ("use_open_telemetry_tracing".equals(key)) {
```

Comment:
This is more than just tracing, so I thought it'd be named more like "use_open_telemetry"